processed 3 tasks

task 1 'run'. lines 13-23:
Error: Script execution failed with VMError: {
    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
    sub_status: Some(1),
    location: script,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 7)],
}

task 2 'run'. lines 25-35:
Error: Script execution failed with VMError: {
    major_status: INTERNAL_TYPE_ERROR,
    sub_status: None,
    location: script,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 7)],
}
